### PR TITLE
SRKVE-876: Update the "Knative Eventing - Broker/Trigger" dashboard info

### PR DESCRIPTION
**Pull request by rh-max**
For versions 4.6+
Jira: https://issues.redhat.com/browse/SRVKE-876
Docs preview: https://deploy-preview-35612--osdocs.netlify.app/openshift-enterprise/latest/serverless/admin_guide/serverless-admin-monitoring?utm_source=github&utm_campaign=bot_dp#serverless-admin-monitoring-eventing-broker-trigger_serverless-admin-monitoring

### CODE REVIEW

File: modules/serverless-admin-monitoring-eventing-broker-trigger.adoc [hunk 1]
```diff
@@ -19,16 +19,17 @@ You can use the {product-title} monitoring dashboards to view metrics for broker
 . Select the *Knative Eventing - Broker/Trigger* dashboard in the *Dashboard* drop-down list.
 . You can now view the following metrics:
 .. For brokers:
-*** Event Count (rate per minute)
-*** Success Rate (2xx Event, rate per minute)
-*** Event Count by Event Type (rate per minute)
-*** Failure Rate (non-2xx Event, fraction of rate per minute)
-*** Event Count by Response Code Class (rate per minute)
+*** Event Count (avg/sec, over 1m window)
+*** Success Rate  (2xx Event, fraction rate, over 1m window)
```
Comment (matzew) [inline modules/serverless-admin-monitoring-eventing-broker-trigger.adoc:23]: do we use two spaces by intention ? or by accident?

File: modules/serverless-admin-monitoring-eventing-broker-trigger.adoc [hunk 2]
```diff
@@ -19,16 +19,17 @@ You can use the {product-title} monitoring dashboards to view metrics for broker
...
+*** Event Count by Response Code Class (avg/sec, over 1m window)
+*** Failure Rate (non-2xx Event, fraction rate, over 1m window)
 *** Event Dispatch Latency (ms)
 .. For triggers:
-*** Event Count (rate per minute)
-*** Success Rate (2xx Event, fraction rate per minute)
-*** Event Count by Response Code Class (rate per minute)
-*** Failure Rate (non-2xx Event, fraction rate per minute)
+*** Event Count (avg/sec, over 1m window)
+*** Success Rate  (2xx Event, fraction rate, over 1m window)
```
Comment (matzew) [inline modules/serverless-admin-monitoring-eventing-broker-trigger.adoc:30]: same w/ the spaces

File: modules/serverless-admin-monitoring-eventing-broker-trigger.adoc (hunk 1 above)
Comment (rh-max) [reply modules/serverless-admin-monitoring-eventing-broker-trigger.adoc]: @matzew Good catch. I copied this from https://github.com/openshift-knative/serverless-operator/pull/1044/files , I left a comment for @skonto regarding it.

**Review by cardil — APPROVED** [review]
/lgtm

from QE

**Review by adellape — COMMENTED** [review]
LGTM, just remember to squash to single commit.

**Review by abrennan89 — APPROVED** [review]
/lgtm

**Pull request merged**